tests/test_cli/test_transfer.py

```diff
@@ -94,6 +101,7 @@ def get_balance(self) -> int:
             wallet = get_wallet_from_agent_config(agent_config, password=self.PASSWORD)
             return int(try_get_balance(agent_config, wallet, self.LEDGER_ID))
 
+    @skip_fetchai_test_macos
     @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS)
```

Comment:
@Karrenbelt Skipped this test

Reply:
yeah we could have skipped on macos 310 alone but its fine I guess.